README.md

```diff
@@ -0,0 +1,54 @@
+# Stonks
+
+> a SPA (single page application) that allows the user to connect to the Financial Modeling Prep API to view Apple, Starbucks, and Ford stock data! A User can navigate through the incoming stock data and navigate their way to the different stocks through various filter pages. Have fun! 
+
+
+## Built With
+- React
+- React Router
+- Redux State Management
+- Financial Modeling Prep API (https://financialmodelingprep.com/developer/docs/)
+- HTML, CSS, Javascript
+
+## Live Demo
+
+Video Walkthrough: https://www.loom.com/share/ab16e33dcc784f32bf2fa2266c377150
+
+Live site: [HERE!](https://akeegandev.github.io/stonks/)
+
+
+## Getting Started
+
+>1) To get started please open your terminal and navigate to whatever directory you house your project repos. 
+>2) Once there copy and paste this SSH key git@github.com:AKeeganDev/stonks.git
+>3) A new Directory will be made. From there you can cd (change directory) to /stonks
+>4) Run $npm install in the root folder directory to download the required packages
+>4) Run $npm start in order run the project and see it in your default browser
+
+
+
+## Authors
+
+👤 **Aaron Keegan**
+
+- GitHub: [@AKeeganDev](https://github.com/AKeeganDev)
+- Twitter: [@AkeeganDev](https://twitter.com/AkeeganDev)
+- LinkedIn: [AKeeganDev](https://linkedin.com/in/AKeeganDev)
+
+## 🤝 Contributing
+Contributions are always welcome. Please open an issue if you have any suggestions or wish to become a collaborator
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Inspiration & Acknowledgments
+
+The inspiration for this project came from my course curriculum at Microverse.
+The design came from Creative Commons. Link to their license [HERE](https://creativecommons.org/licenses/by-nc/4.0/legalcode)
+The study of JavaScript, React, Webpack usage, Axios, and API usage made this a perfect project for practice.
```

Comment:
- [OPTIONAL] Please credit [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) as requried by the [Creative Commons License](https://creativecommons.org/licenses/)